### PR TITLE
Adds more medibot skins

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
* Adds more medibot skins

## Why It's Good For The Game
consistent appearance is good
also, it'll be a good implementation when we're going to give different ability per its appearance.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

sorry for my spriting skill if these look bad, but it can be changed someday...

</details>

## Changelog
:cl:
tweak: medibots have more skins based on which first aid kit was used when it's made. These currently have no speicla ability by its shape which means these are just skins.
imageadd: more medibot skins (advanced, rad, syndi, doctors roundstart kit)
/:cl:

### CODE REVIEW

File: code/__DEFINES/bot_defines.dm [hunk 1]
```diff
@@ -3,4 +3,15 @@
 #define BOT_Z_MODE_SUMMONED 30
 
 #define MULEBOT_Z_MODE_ACTIVE 10
- 
+
+
+
+#define MEDBOT_SKIN_BRUTE "brute"
```
Comment (Penwin0) [inline code/__DEFINES/bot_defines.dm:11]: Add a comment

**Review by Penwin0 — DISMISSED** [review]
I like it, looks good

File: code/__DEFINES/bot_defines.dm (hunk 1 above)
Comment (EvilDragonfiend) [reply code/__DEFINES/bot_defines.dm:11]: added

File: code/game/objects/items/storage/firstaid.dm [hunk 2]
```diff
@@ -373,15 +373,33 @@
 		to_chat(user, "<span class='warning'>You need to empty [src] out first!</span>")
 		return
 
+	var/skin_type = null
+	switch(src.icon_state)
+		if("firstaid","firstaid-old")
+			skin_type = null
+		if("firstaid-brute","firstaid-brutealt")
+			skin_type = MEDBOT_SKIN_BRUTE
```
Comment (itsmeow) [inline code/game/objects/items/storage/firstaid.dm]: add a field to all medkits, called medibot_skin or whatever instead

**Review by itsmeow — CHANGES_REQUESTED** [review]
pretty epic

**Comment by EvilDragonfiend** [comment]
changed

File: code/game/objects/items/storage/firstaid.dm [hunk 3]
```diff
@@ -373,32 +385,12 @@
...
-			skin_type = MEDBOT_SKIN_ADVANCED
-		if("firstaid-rad","firstaid-radalt")
-			skin_type = MEDBOT_SKIN_RADIATION
-		if("firstaid-combat","firstaid-combatalt")
-			skin_type = MEDBOT_SKIN_SYNDI
-		else
-			to_chat(user, "<span class='warning'>[src] seems to be incompatible to make a medibot!</span>")
-			return
+	if(!src.skin_type)
+		to_chat(user, "<span class='warning'>[src] seems to be incompatible to make a medibot!</span>")
```
Comment (itsmeow) [inline code/game/objects/items/storage/firstaid.dm]: [src] cannot be used to make a medibot!

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Review by itsmeow — APPROVED** [review]

**Pull request merged**